### PR TITLE
Fix translators issue

### PR DESCRIPTION
**Pull request by kylinroc**
Make post header look like this:

screen shot 2018-10-30 at 1 07 58 am

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -29,7 +29,19 @@ <h2>
...
+      {% for trans in page.translators %}
+          {% if forloop.last %}&{% endif %}
+          {% assign translator = site.translators | where:"name",trans | first %}
+          {% if translator %}
+            <a class="translator" href="{{ translator.url }}">{{ translator.name }}</a>
+          {% else %}
+            {{ trans }}
+          {% endif %}
+      {% endfor %}
+      - <wbr/>
```
Comment (mattt) [inline _layouts/post.html]:
```suggestion
      — <wbr/>
```

Comment (mattt) [reply _layouts/post.html]: Replace hyphen-minus sign with an em dash (—, U+2024)

File: _layouts/post.html [hunk 2]
```diff
@@ -29,7 +29,19 @@ <h2>
     </span>
 
     <span>
-    {% if translator %}
+    {% if page.translators %}
+      {% t translated_by %}
+      {% for trans in page.translators %}
+          {% if forloop.last %}&{% endif %}
```
Comment (mattt) [inline _layouts/post.html]:
```suggestion
          {% if forloop.last %}&amp;{% endif %}
```

Comment (mattt) [reply _layouts/post.html]: Properly escape ampersand

**Comment by kylinroc** [comment]
@mattt Done.

**Comment by mattt** [comment]
Thanks again for this, @kylinroc!

**Pull request merged**